content/rs/security/certificates/updating-certificates.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Update certificates
+linkTitle: Update certificates
+description: Update certificates in a Redis Enterprise cluster.
+weight: 20
+alwaysopen: false
+categories: ["RS"]
+aliases: ["/rs/administering/cluster-operations/updating-certificates",
+"/rs/administering/cluster-operations/updating-certificates/",
+"/rs/administering/cluster-operations/updating-certificates.md"]
+---
+
+{{<warning>}}
+When you update the certificates, the new certificate replaces the same certificates on all nodes in the cluster.
+{{</warning>}}
+
+## How to update certificates
+
+You can use either the rladmin command-line interface (CLI) or the REST API to update certificates.
+
+The new certificates are used the next time the clients connect to the database.
+
+When you upgrade Redis Enterprise Software, the upgrade process copies the certificates that are on the first upgraded node to all of the nodes in the cluster.
+
+### Use the CLI
+
+To replace certificates using the rladmin CLI, run:
+
+```sh
+ rladmin cluster certificate set <cert-name> certificate_file <cert-file-name>.pem key_file <key-file-name>.pem
+```
+
+Where:
+
+- cert-name - The name of the certificate you want to replace:
```

Comment:
```suggestion
- `<cert-name>` - The name of the certificate you want to replace:
```

Reply:
I added these to help the variables stand out a bit more and make it easier to scan. But it's a stylistic choice that's up to you.